### PR TITLE
Add crypto mining pool scam section to /security [Fixes #4655]

### LINKED ISSUE #4655: Add section on crypto mining pool scams
**Is your feature request related to a problem? Please describe.**

Crypto mining pool scams seem to be the newest popular attack vector for scamming people out of their crypto. We should add this to our security page (and try to optimise for ranking for the term @samajammin ?)

**Describe the solution you'd like**

Add a section on crypto mining pool scams here on our [security and scam prevention page](https://ethereum.org/en/security/)

**Additional context**

https://www.indiatimes.com/technology/news/crypto-mining-pool-scam-on-twitter-555251.html
https://www.reddit.com/r/CoinBase/comments/r0qe0e/scam_or_possible_incredible_payout/
https://www.reddit.com/r/CryptoCurrency/comments/r2atqi/im_the_guy_from_a_few_days_ago_that_got_scammed/

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by joneskj55**
## Description
This PR adds a section about crypto mining pool scams to the /security page.

## Related Issue
#4655

### CODE REVIEW

**Review by corwintines — APPROVED** [review]
Awesome, thank you @joneskj55!

**Comment by joneskj55** [comment]
@corwintines Anytime, thanks!

File: src/content/security/index.md [hunk 1]
```diff
@@ -232,6 +232,20 @@ These fraudulent brokers find their targets by using fake accounts on YouTube to
 
 ![A trading broker scam on YouTube](./brokerScam.png)
 
+### Crypto mining pool scams {#mining-pool-scams}
+
+Mining pool scams involve people contacting you unsolicited, and claiming that you can make large returns by joining an Ethereum mining pool. The scammer will make claims and stay in contact with you for however long it takes. Essentially, the scammer will try and convince you that when you join an Ethereum mining pool, your cryptocurrency will be used to create Ethereum and that you will be paid dividends in the form of Ethereum. What ends up happening is, you will notice that your cryptocurrency is making small returns. This is simply to bait you into investing more. Eventually, all of your funds will be sent to an unknown address and the scammer will either disappear or in some cases will continue to stay in touch as has happened in a recent case.
```
Comment (nassersaazi) [inline src/content/security/index.md:237]:
```suggestion
Mining pool scams involve people contacting you unsolicited, and claiming that you can make large returns by joining an Ethereum mining pool. The scammer will make claims and stay in contact with you for however long it takes. Essentially, the scammer will try and convince you that when you join an Ethereum mining pool, your cryptocurrency will be used to create Eth and that you will be paid dividends in the form of Eth. What will end up happening is, you will notice that your cryptocurrency is making small returns. This is simply to bait you into investing more. Eventually, all of your funds will be sent to an unknown address and the scammer will either disappear or in some cases will continue to stay in touch as has happened in a recent case.
```

Comment (nassersaazi) [reply src/content/security/index.md:237]: just thought there should be a distinction between Ethereum the blockchain and Eth the asset.

Comment (joneskj55) [reply src/content/security/index.md:237]: Hi @nassersaazi I think you are right, thanks! Would you mind adding a commit that changes Eth to ETH please first?

Comment (nassersaazi) [reply src/content/security/index.md:237]: alright

**Comment by joneskj55** [comment]
@nassersaazi Thanks for the assist!

**Review by wackerow — APPROVED** [review]
LGTM! Thanks @joneskj55!

**Comment by wackerow** [comment]
@all-contributors please add @joneskj55 for content

**Pull request merged**